packages/css-modules-flow-types-printer/index.js

```diff
@@ -3,10 +3,9 @@
 import os from 'os';
 
 export default function printFlowDefinition(tokens, indent = '  ') {
-  const props = Object.keys(tokens)
-    .sort()
-    .map(key => `${indent}+'${key}': string;`)
-    .join(os.EOL);
+  const props = tokens && tokens.length
```

Comment:
Let us assume that you always receives an array here. Maybe name the parameter `tokensArray` for clarifying this.